gmail/Package.md

```diff
@@ -10,7 +10,7 @@ Ballerina Gmail Connector provides the capability to send, read, and delete emai
 
 |                       | Version                       |
 |-----------------------|-------------------------------|
-| Ballerina Language    | Swan Lake 2201.0.0            |
+| Ballerina Language    | Swan Lake 2201.0.2            |
```

Comment:
Latest language release is 2201.0.3. So don't we need to keep this with latest version ?

Reply:
Updated